src/editors/containers/ProblemEditor/components/SelectTypeModal/content/Preview.test.tsx

```diff
@@ -30,7 +65,7 @@ describe('Preview', () => {
     expect(screen.getByText('Example Title problem')).toBeInTheDocument();
     expect(screen.getByText('Example description')).toBeInTheDocument();
     expect(screen.getByRole('img')).toHaveAttribute('src', 'example.png');
-    expect(screen.getByRole('img')).toHaveAttribute('alt', 'A preview illustration of a null problem');
+    expect(screen.getByRole('img')).toHaveAttribute('alt', 'A preview illustration of a example problem');
```

Comment:
To fix this so it doesn't say "null", you need to update the message string itself, here: https://github.com/openedx/frontend-app-authoring/blob/11de4022f0ca86f0991c6f4cdcf612497c7a24b8/src/editors/containers/ProblemEditor/components/SelectTypeModal/content/messages.ts#L61

But I think there's an even better way to handle this sort of thing - see my previous suggestion.